docs/book/installation.rst

```diff
@@ -74,8 +95,41 @@ browser.
 
 .. tip::
 
-    You can also use the Symfony built-in webserver by running the ``php app/console server:run --env dev``
-    command and then accessing ``http://localhost:8000``.
+    We strongly recommend using the Symfony abuilt-in webserver by running the
+    ``php app/console server:run --env dev``
+    command and then accessing ``http://localhost:8000``
+
+.. note::
+
+    The localhost's 8000 port may be already occupied by some other process.
+    If so you should try other ports, like for instance:
+    ``php app/console server:run localhost:8081 --env dev``
```

Comment:
What if we leave this one and add the one you've suggested? I think it is a common mistake while using built in server for devs that are newbies :)